src/app/brukerprofil/adresse/midlertidigAdresseNorge/MidlertidigAdresseNorge.tsx

```diff
@@ -153,7 +154,7 @@ class MidlertidigAdresseNorge extends React.Component<Props> {
         return (
             <>
                 <Select
-                    label="Landkode"
+                    label="Type adressee"
```

Comment:
En «e» for mye her. Og hadde kanskje vært bedre med «Adressetype»?

Reply:
enig!